### PR TITLE
fix: bump all-contributors-cli to v6.14.2

### PR DESCRIPTION
**Pull request by baikho**
Fix for https://github.com/all-contributors/all-contributors/issues/381

### CODE REVIEW

**Comment by baikho** [comment]
Isn't that still covered with the same major release of `6.x.x`? My local already installs me `6.14.2` in `package-lock.json`.

CircleCI error mentions:

```
yarn install v1.5.1
[1/4] Resolving packages...
[2/4] Fetching packages...
error chalk@4.0.0: The engine "node" is incompatible with this module. Expected version ">=10".
error An unexpected error occurred: "Found incompatible module".
```
When I check my local I'm running node `v11.15.0`, but CircleCI seems to use a docker image of node `v8.10.0`:

```
docker_defaults: &docker_defaults
  docker:
    - image: circleci/node:8.10.0
```

Is it ok to update this?

**Comment by Berkmann18** [comment]
Yes, as long as the Node.js version used by CircleCI satisfies the constraints of the dependencies, so in this case `chalk`.

**Comment by Berkmann18** [comment]
@all-contributors Please add @baikho for maintenance

File: .circleci/config.yml [hunk 1]
```diff
@@ -2,7 +2,7 @@ version: 2.1
 
 docker_defaults: &docker_defaults
   docker:
-    - image: circleci/node:8.10.0
+    - image: circleci/node:11.15.0
```
Comment (jakebolam) [inline .circleci/config.yml:5]: Any reason for changing the version? We run version 8 in production: https://github.com/all-contributors/all-contributors-bot/blob/master/serverless.yml#L33

**Comment by jakebolam** [comment]
Thank you @baikho, this is great

File: .circleci/config.yml (hunk 1 above)
Comment (baikho) [reply .circleci/config.yml:5]: yes, for `chalk 4` requiring `node >=10` it appears

**Comment by Berkmann18** [comment]
It would be nice to get this merged (I can't do that) to see where recent regressions are occurring.

File: .circleci/config.yml (hunk 1 above)
Comment (Berkmann18) [reply .circleci/config.yml:5]: @jakebolam https://github.com/all-contributors/all-contributors-bot/pull/305#pullrequestreview-400516419

**Comment by gr2m** [comment]
this is done via #330

**Pull request closed without merging**